README.md

```diff
@@ -54,6 +54,8 @@ Specifies the value for the `Domain` `Set-Cookie` attribute. By default, no doma
 is set, and most clients will consider the cookie to apply to only the current
 domain.
 
+**Note:** Some browsers will not set a cookie on `localhost` is the `domain` attribute is set, so you should omit this setting when testing on `localhost`.
```

Comment:
Please make a separate PR for this note, as it is not related to "info about implementing store.js methods"